kms-message/test/test_kms_request.c

```diff
@@ -964,6 +937,83 @@ kms_response_parser_test (void)
    kms_response_parser_destroy (parser);
 }
 
+typedef struct {
+   const char *filepath;
+   const char *expected_body;
+   int max_to_read;
+   int expected_status;
+} parser_testcase_t;
+
+/* File should have \r\n line endings (use /etc/rewrite.py) */
```

Comment:
Who is supposed to call rewrite.py? You can use git attributes to control how the file is checked out - https://www.git-scm.com/docs/gitattributes

Reply:
It's in etc, so just a development script for producing `\r\n` on non Windows. The default seems to preserve the `\r\n` (at least for .bin files).